lib/inspec/schema/exec_json.rb

```diff
@@ -1,4 +1,4 @@
-require_relative "./primitives"
+require "lib/inspec/schema/primitives"
```

Comment:
no, not "lib". lib is already in the load path (via rake/gemfile/etc)... this would only work if "." were in the load path as well, which it isn't by default.

Reply:
Resolved as of https://github.com/inspec/inspec/pull/4330/commits/6dd9b4d21f8db0c4d8180795c6059f6fedb08d31.

Seemed to be working either way but yes, this is nicer.